src/platform/unix/mod.rs

```diff
@@ -649,10 +647,67 @@ fn make_socket_lingering(sockfd: c_int) -> Result<(),UnixError> {
     Ok(())
 }
 
+struct BackingStore {
+    fd: c_int
+}
+
+impl BackingStore {
+    pub fn new(length: usize) -> BackingStore {
+        let count = GLOBAL_SHM_COUNT.fetch_add(1, Ordering::SeqCst);
```

Comment:
Haha. Yeah, I always forget that Relaxed is the same as LLVM's definition of monotomic.